spec/swagger.yaml

```diff
@@ -1717,30 +1727,35 @@ paths:
         - name: projectId
           in: path
           required: true
+          description: Unique identifier of a project
           type: string
           format: uuid
         - name: content
-          description: The context file part with content type. Supported content types are image/\* and text/html.
           in: formData
           required: true
           type: file
+          description: |-
+            The context file part with content type. Supported content type is `text/html`.
         - name: name
           in: formData
           required: false
           type: string
-          description: Optional parameter that overrides the file name of *content* part.
+          description: |-
+            Context URL or whatever has to be served as context name.
+            Overrides the filename in `content` part.
         - name: matchParams
           in: formData
+          description: |-
+            Array of string hashcodes to match against, and / or context override specifier.
+            If string hashcodes not specified, *all strings from the project will be included in match*.
+            If context override not specified, *strings with context will not be bound to this context*.
+            
+            See parameter type description for `/context-api/v2/projects/{projectId}/contexts/{contextUid}/match/async`
           required: false
           type: string
-          description: A part containing JSON object with an array of string hashcodes to match against.
-                       If the part content is missing, or empty array is passed, *all strings in the project* will be included in the match.
-                       This part contains the same parameters as a body of `/projects/{projectId}/contexts/{contextUid}/match/async` request.
       responses:
-        200:
-          description: OK
         202:
-          description: ACCEPTED
+          description: Request Accepted and processing started
```

Comment:
The description for 202 should be 'ACCEPTED' so it is consistent with other endpoints.

The asynchronous manner of the endpoint should be described in the endpoint description. See Jobs API for examples

Reply:
UPDATED